src/main/java/model/events/MessageSent.java

```diff
@@ -2,14 +2,23 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import model.InboundMessage;
+import model.Initiator;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 
 public class MessageSent {
 
     private InboundMessage message;
 
+    private Initiator initiator;
 
+    public Initiator getInitiator() {
```

Comment:
Since same code is duplicated across several classes, I would suggest having a base abstract class containing the Initiator field and the corresponding getter and setter.